conjure-java-jaxrs-client/src/main/java/feign/QosErrorDecoder.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feign;
```

Comment:
Why is this in `feign` package?

Reply:
This is in the feign package because it implements the feign interface `ErrorDecoder`. Other classes in the feign package include [`ConjureCborDelegateDecoder`](https://github.com/palantir/conjure-java-runtime/blob/8d27d3fdf8ef09f2f0c65d35b659dcdbff85c89c/conjure-java-jaxrs-client/src/main/java/feign/ConjureCborDelegateDecoder.java#L39).

Reply:
Some of those classes use feign internal utilities which we should not depend on. We should not put our classes into other organizations packages.
See our style guide:
https://github.com/palantir/gradle-baseline/blob/develop/docs/java-style-guide/readme.md#package-statement
> The package name of internal as well as open source projects starts with com.palantir.<project name>